modules/contact/test/tests/3d-mortar-contact/tests

```diff
@@ -39,7 +46,7 @@
     mesh_mode = 'REPLICATED'
```

Comment:
Is this a VCP test? If so, please add a comment (sorry I know this isn't from your PR)

Reply:
Will do - No problem, I am in the middle of addressing several remaining comments from you and Antonio. 

I'll ping you and Antonio when the PR is ready to spare you from endless github messages :)

Reply:
> Is this a VCP test? If so, please add a comment (sorry I know this isn't from your PR)

No it is not. Let me remove the `Replicated` mesh mode specification here